runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpServerFactory.java

```diff
@@ -6655,6 +6643,19 @@ private static Array32FW<HttpHeaderFW> initHeaders(
         return builder.build();
     }
 
+    private static Array32FW<HttpHeaderFW> initHeadersEmpty(
+            HttpConfiguration config,
+            String16FW status)
+    {
+        final Array32FW.Builder<HttpHeaderFW.Builder, HttpHeaderFW> builder =
+                new Array32FW.Builder<>(new HttpHeaderFW.Builder(), new HttpHeaderFW())
+                        .wrap(new UnsafeBuffer(new byte[64]), 0, 64)
+                        .item(h -> h.name(HEADER_STATUS).value(status));
+
```

Comment:
Let's include the `server` header when explicitly specified, same as `initHeaders`.